app/handler/trades.js

```diff
@@ -377,6 +377,7 @@ exports.createOffer = function (details, callback) {
                             } else if (err.message.indexOf('can only be sent to friends') != -1) {
                                 return callback(err);
                             } else if (err.message.indexOf('maximum number of items allowed in your Team Fortress 2 inventory') > -1) {
+                                admin.message('Alert! My inventory is or almost full. Please do something.');
```

Comment:
"inventory is or almost full", doesn't seem like correct english

Also check for this when loading the inventory, not just when an offer failed

Reply:
I'd suggest you'd change it something like "My inventory is full or almost full, please take action.`` Something around those lines.